manifests/3.0.0/opensearch-3.0.0.yml

```diff
@@ -227,3 +227,9 @@ components:
     platforms:
       - linux
       - windows
+  - name: skills
```

Comment:
A lot of information missing here, please check 2.17 manifest I think all these depends_on are missed:
```

    depends_on:
      - job-scheduler
      - anomaly-detection
      - sql
      - ml-commons
 ```

Reply:
Yes. `depends_on` is used for our incremental build to know the plugin dependencies. Any rebuild on dependency plugin will trigger rebuild on the `skills` plugin.

Reply:
Ack. Updated. Sorry, this is the first time i'm raising PR to this repo.